tools/Google.Cloud.Tools.Analyzers.Tests/DefaultLiteralTest.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TestHelper;
+using Xunit;
+
+namespace Google.Cloud.Tools.Analyzers.Tests
+{
+    public class DefaultLiteralTest : CodeFixVerifier
+    {
+        [Fact]
```

Comment:
Possibly add tests to verify it doesn't trigger for non-nullable versions?

Reply:
Done